src/test/java/core/basesyntax/service/ReportServiceTest.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceTest {
+
+    private static final ReportService REPORT_SERVICE = new ReportServiceImpl();
```

Comment:
in all test classes:
```suggestion
    private final ReportService reportService = new ReportServiceImpl();
```
1. There is no need to make service static
2. Service is not an immutable class so cannot be proper constant with UPPERCASE_NAMING